vcs/vcsvtk/fillareautils.py

```diff
@@ -63,7 +63,8 @@ def computeMarkerScale(dataRange, screenGeom, pxScale=None):
     # screenAspect = float(screenGeom[0]) / screenGeom[1]
 
     scale = max(*diffwpoints)
-    if dataAspect > 1:
+    print("DA")
+    if dataAspect < 1:
```

Comment:
@doutriaux1 @danlipsa This is the change that is causing the changes in marker size and the "black screen" (which just really huge markers) on this branch.  This must have slipped in somehow from our experimentation while I was visiting.

As @doutriaux1 and I discussed a couple weeks ago, I'm going to revisit the handling of marker size again soon enough anyway, but I'll push a commit removing this and then create a `vtk_master` branch on the baseline repo and see if we can't get this though so we can really start taking @danlipsa latest VTK build.

Reply:
Good you found it. Not sure why I have not seen this. I was testing with master - maybe I did not pull the latest, or was this on a branch?